pkgs/applications/networking/seaweedfs/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, buildGoModule }:
+
+buildGoModule rec {
+  pname = "seaweedfs";
+  version = "2.07";
+
+  src = fetchFromGitHub {
+    owner = "chrislusf";
+    repo = "seaweedfs";
+    rev = "${version}";
+    sha256 = "0yrfpj4ph9f321vqfn1zadv44pqa3ivjq9rx6gsz9hlv50gfaqn1";
+  };
+
+  vendorSha256 = "1ysqagi4y25bi84h5fhkdimnsigy43klf6hrcqn7q75382fb4bzn";
+
+  subPackages = [ "weed" ];
+
+  meta = with lib; {
+    description = "Simple and highly scalable distributed file system";
+    homepage = "https://github.com/chrislusf/seaweedfs";
+    license = licenses.asl20;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
I am not 100% sure here but I think this defaults to the platforms go supports and should only be changed if not all of those are supported.

Reply:
looks like you're right (https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/go-modules/generic/default.nix#L244), removed